pcsx2/USB/usb-pad/usb-pad.cpp

```diff
@@ -439,17 +444,25 @@ namespace usb_pad
 		}
 	}
 
+	s16 PadState::applySteeringAxisDeadzone(float value)
```

Comment:
Try to keep to the same naming convention as the surrounding functions, i.e. title case / `ApplySteeringAxisDeadzone()`.